MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -671,6 +671,9 @@ protected virtual void CreateDeviceResources()
 
             // Get Direct3D 11.1 context
             _d3dContext = _d3dDevice.ImmediateContext.QueryInterface<SharpDX.Direct3D11.DeviceContext>();
+            
+            // Create a new instance of GraphicsDebug because we support it on Windows platforms.
+            _graphicsDebug = new GraphicsDebug(this);
```

Comment:
This should be moved to the common non-platform specific code if we really plan to support it across all platforms.  It is better than the developer having to check for `null` on `GraphicsDebug`.